include/ximea_driver/ximea_driver.hpp

```diff
@@ -0,0 +1,24 @@
+#pragma once
+
+#include <memory>
+
+#include "m3api/xiApi.h"
```

Comment:
is the ximea campera api expected to already be installed, or should this be part of the build file?

Reply:
You can install it using `tools/install_sdk.sh`